ctapipe/reco/HillasReconstructor.py

```diff
@@ -193,8 +196,8 @@ def inititialize_hillas_planes(
 
         self.hillas_planes = {}
         for tel_id, moments in hillas_dict.items():
-            p2_x = moments.x + 0.1 * u.m * np.cos(moments.psi)
-            p2_y = moments.y + 0.1 * u.m * np.sin(moments.psi)
+            p2_x = moments.x + moments.length * np.cos(moments.psi)
+            p2_y = moments.y + moments.length * np.sin(moments.psi)
```

Comment:
Only the direction of the vector is of use here, that's why its length was set to an arbitrary value of 0.1*u.m.
Does `moments.length` carries more information here?
Either way, it is fine, unless:
- `moments.length` has a 0 or `nan` value (but not psi which is really what matters here)
- `p2` is used for graphical representation of the direction, in which case `moments.length` might be small in many cases. But that could be that problem could be left to the visualization.

Reply:
>     * `p2` is used for graphical representation of the direction, in which case `moments.length` might be small in many cases. But that could be that problem could be left to the visualization.

good point. maybe leaving a constant 0.1 here makes more sense.

Reply:
The vectors are weighted with `intensity * length / width` so having a `nan` or `0` length will in any case have a invalid reconstruction.

But I can put the `0.1 m` back. I was worried that it is camera dependent and could be too large for small cameras. Might this be the case? Or does it not matter?

Reply:
I added the 0.1 meters back